package.json

```diff
@@ -109,4 +109,4 @@
         "rollup-plugin-json": "^4.0.0",
         "typescript": "^3.4.1"
     }
-}
+}
```

Comment:
could you add the missing EOF :)

Reply:
Okay sure

Reply:
I think it's there. no need to add brace.

Reply:
```suggestion
}

```

Reply:
Ya, I know this is to be fixed. thanks, @Levino.